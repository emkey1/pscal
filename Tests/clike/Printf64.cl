int main() {
    int i;
    float d;
    i = 42;
    d = 3.25;
    printf("%lld %lf\n", i, d);
    return 0;
}

