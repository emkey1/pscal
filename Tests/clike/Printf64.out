42 3.250000
